packages/shared/lib/services/account.service.ts

```diff
@@ -111,4 +112,84 @@ class AccountService {
     }
 }
 
+function parseTeamName({ name, email }: { name: string; email?: string | undefined }): string {
+    if (!email) {
+        return `${name}'s Team`;
+    }
+
+    const freeEmailDomains = [
```

Comment:
could be stored outside never too sure what's best when a function is not called often

Reply:
updated